src/ui/Pricing/Page/BlackFridayBanner.svelte

```diff
@@ -0,0 +1,80 @@
+<script>
+  import blackFriday from './blackFriday.png'
+
+  let className = ''
+  export { className as class }
+</script>
+
+<section class="row justify v-center border body-2 relative {className}">
+  <div class="text">
+    <div class="title">Buy this bear market dip</div>
+    <div class="value">35% Off</div>
+    <div class="condition">for all subscriptions till Nov 30th</div>
+    <a class="btn btn-1 body-1 mrg--b mrg-l" href="https://santiment.net/discord"
+      >Claim your promo code!</a
+    >
+    <div class="note">You will find your promo code in our dicord channel</div>
+  </div>
```

Comment:
Use semantic tags

Reply:
What do you suggest for them? h1...h6 ?

Reply:
Use the ones you think fit the best. I will review once it's ready